### PR TITLE
Basic Gambit API integration

### PR DESCRIPTION
**Pull request by sergiitk**
#### What's this PR do?

Implements Campaign endpoints of [Gambit API](https://github.com/DoSomething/gambit/tree/develop/documentation):
- [`GET /v1/campaigns`](http://ds-mdata-responder-staging.herokuapp.com/v1/campaigns): `Gambit::getAllCampaigns()`
- [`GET /v1/campaigns/:id`](http://ds-mdata-responder-staging.herokuapp.com/v1/campaigns/4944): `Gambit:: getCampaign($id)`
- Covers API with corresponding unit tests using Mock objects
#### Any background context you want to provide?

Current Gambit response are yet to be normalized, see DoSomething/gambit#681.
#### Todo

Todo: `POST /v1/chatbot`
#### Where should the reviewer start?
- @weerd Please review integration with Gateway
- @aaronschachter Please review integration with Gambit
- @deadlybutter CC
#### What are the relevant tickets?

Ref DoSomething/mbc-registration-email#71.

### CODE REVIEW

**Comment by weerd** [comment]
@sergii-tkachenko I added @DFurnes as well :)

**Comment by sergiitk** [comment]
Yes, thanks, @weerd!

File: tests/Helpers/Gambit/CampaignResponse.php [hunk 1]
```diff
@@ -0,0 +1,31 @@
...
+use DoSomething\GatewayTests\Helpers\JsonResponse;
+
+class CampaignResponse extends JsonResponse
+{
+    /**
+     * Make a new mock Gambit campaign by id response.
+     */
+    public function __construct($code = 200)
+    {
+        $body = ['data' => [
```
Comment (DFurnes) [inline tests/Helpers/Gambit/CampaignResponse.php]: Can you adjust this indentation?

``` php
$body = [
    'data' => [
        'id' => 4944,
        // ...
    ],
];

```

File: tests/Helpers/Gambit/CampaignsResponse.php [hunk 2]
```diff
@@ -0,0 +1,46 @@
...
+use DoSomething\GatewayTests\Helpers\JsonResponse;
+
+class CampaignsResponse extends JsonResponse
+{
+    /**
+     * Make a new mock Gambit list all campaigns response.
+     */
+    public function __construct($code = 200)
+    {
+        $body = ['data' => [
```
Comment (DFurnes) [inline tests/Helpers/Gambit/CampaignsResponse.php]: Same. 👾

**Review by DFurnes — COMMENTED** [review]
Two nitpicky style requests, otherwise looks great to me! 👍

File: tests/Helpers/Gambit/CampaignResponse.php (hunk 1 above)
Comment (sergiitk) [reply tests/Helpers/Gambit/CampaignResponse.php]: Actually, made this on purpose, thought it'd be more readable.
But ok, I always respect standards 😉 .

**Comment by sergiitk** [comment]
@DFurnes Fixed. Not exactly as in your example, I didn't want it to be tabbed 3 times.

**Comment by DFurnes** [comment]
I'm a little confused why you're avoiding indentation, but it's not a big deal. Let's ship it. 🚢

**Review by DFurnes — APPROVED** [review]

**Review by weerd — APPROVED** [review]
Looks good! Ditto @DFurnes... @sergii-tkachenko not sure why indentation in tests going 3 places is weird.  Just the standard we set forth from other tests and for readability, but not a huge deal. 👍

File: tests/Helpers/Gambit/CampaignsResponse.php [hunk 3]
```diff
@@ -0,0 +1,44 @@
...
+    {
+        $body = [];
+        $body['data'][] = [
+            'id' => 46,
+            'rb_noun' => 'Vampires',
+            'rb_verb' => 'Unplugged',
+            'msg_rb_confirmation' => 'You SLAYED this campaign. Badass. ',
+            'title' => 'Don\'t Be a Sucker',
+            'tagline' => 'Unplug unused electronics to conserve energy.',
+            'status' => '6196',
```
Comment (aaronschachter) [inline tests/Helpers/Gambit/CampaignsResponse.php]: This should be `active` or `closed`

**Comment by sergiitk** [comment]
@DFurnes Trying to fit into 120 chars limit.

File: tests/Helpers/Gambit/CampaignsResponse.php (hunk 3 above)
Comment (sergiitk) [reply tests/Helpers/Gambit/CampaignsResponse.php]: Ah, good point. Thanks.

**Pull request merged**